src/main/scala/net/cakesolutions/CakeStandardsPlugin.scala

```diff
@@ -46,6 +46,8 @@ object CakeStandardsPlugin extends AutoPlugin {
             "-Xlint",
             "-Yno-adapted-args",
             "-Ywarn-dead-code",
+            "-Ywarn-value-discard",
```

Comment:
I'm pretty sure this fires up a false positive for every use of a scalatest matcher.

There is a similar, but superior, check included in wartremover which is disabled for testing.